packages/deployment/app/config.yml

```diff
@@ -95,6 +95,70 @@ default:
 
   # set to true to activate pointInTime on all tables
   enablePointInTime: false
+  tables:
+    # REVIEW: Autoscaling policy configuration is the same for all tables, to
```

Comment:
Is there a way that we could use some sort of sensible default for all tables and then override one-by-one?  Seems like a ton of boilerplate to say the same thing over and over and makes adding a new table more cumbersome.